odoo/models.py

```diff
@@ -5101,8 +5102,11 @@ def _load_records(self, data_list, update=False):
                 to_create.append(data)
 
         # update existing records
-        for data in to_update:
-            data['record']._load_records_write(data['values'])
+        if ignore_duplicates:
+            data_list = [data for data in data_list if data not in to_update]
```

Comment:
why do you remove records you don't update from data_list ? 

where/what for is used the result of this function? :thinking:

Reply:
I removed them because the `data_list` is used to construct the `created_records` of `chart_template.py`
What I understand is that, if I need to ignore duplicates, these ignored values should not be considered in the created_records.